streaming/streaming.go

```diff
@@ -73,15 +73,15 @@ type DstConnCfg struct {
 }
 
 type DatastreamCfg struct {
-	StreamId                    string     `json:"streamId"`
-	StreamLocation              string     `json:"streamLocation"`
-	StreamDisplayName           string     `json:"streamDisplayName"`
-	SourceConnectionConfig      SrcConnCfg `json:"sourceConnectionConfig"`
-	DestinationConnectionConfig DstConnCfg `json:"destinationConnectionConfig"`
-	Properties                  string     `json:"properties"`
-	TableList                   []string   `json:"tableList"`
-	MaxConcurrentBackfillTasks  string     `json:"maxConcurrentBackfillTasks"`
-	MaxConcurrentCdcTasks       string     `json:"maxConcurrentCdcTasks"`
+	StreamId                    string              `json:"streamId"`
+	StreamLocation              string              `json:"streamLocation"`
+	StreamDisplayName           string              `json:"streamDisplayName"`
+	SourceConnectionConfig      SrcConnCfg          `json:"sourceConnectionConfig"`
+	DestinationConnectionConfig DstConnCfg          `json:"destinationConnectionConfig"`
+	Properties                  string              `json:"properties"`
+	TableSchemaMap              map[string][]string `json:"tableList"`
```

Comment:
Is this backward compatible? Will this cause existing streaming JSON configurations to break?

Reply:
It is backward compatible

Reply:
This is part of `streamingCfg` which is used exposed to the customer in case of a non-sharded migration right?